File: Server/Server/Entities/Task.cs
namespace Server.Entities
{
    public class Task
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public DateTime DueDate { get; set; }
        public int Priority { get; set; }
        public bool Completed { get; set; }
        public int UserID { get; set; }
        public string? PhotoUrl { get; set; }
        public double? Latitude { get; set; }
        public double? Longitude { get; set; }
    }
}

File: AndroidClient/app/src/main/java/com/example/androidclient/todo/data/Task.kt
package com.example.androidclient.todo.data

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "tasks")
data class Task(
    @PrimaryKey val id: Int = -1,
    val title: String = "",
    val description: String = "",
    val dueDate: String = "",
    val priority: Int = 0,
    val completed: Boolean = false,
    val photoUrl: String? = null,
    val latitude: Double? = null,
    val longitude: Double? = null,
    val pending: Boolean = false
)